lib/src/wallet_connect/session.dart

```diff
@@ -362,16 +375,16 @@ class WalletConnection extends ValueListenable<WalletConnectState> {
       "addr": addressStr
     };
 
-    final converter = const JsonEncoder()
-        .fuse(const Utf8Encoder())
-        .fuse(const Base64Encoder());
+    final base64Converter = Base64UrlEncoder();
+    final converter =
+        const JsonEncoder().fuse(const Utf8Encoder()).fuse(base64Converter);
```

Comment:
Did you consider `base64Url` like `const JsonEncoder().fuse(const Utf8Encoder()).fuse(base64Url)`? Not sure if that is different.
https://stackoverflow.com/questions/56201074/how-to-encode-and-decode-base64-and-base64url-in-flutter-dart

Reply:
i did consider chaining them, but i use the json to base64 encoded elsewhere and i did not see a reason to create a second instance since they do not hold state.

Reply:
both methods appear to do the same thing. They method leave trailing '='